src/main/java/uk/gov/ons/census/casesvc/service/QuestionnaireLinkedService.java

```diff
@@ -45,7 +45,8 @@ public void processQuestionnaireLinked(
     Case caze = caseService.getCaseByCaseId(UUID.fromString(uac.getCaseId()));
 
     if (isIndividualQuestionnaireType(questionnaireId) && caze.getCaseType().equals("HH")) {
-      caze = caseService.prepareIndividualResponseCaseFromParentCase(caze);
+      caze = caseService.prepareIndividualResponseCaseFromParentCase(caze, UUID.randomUUID());
```

Comment:
More of a style comment, but is it worth giving a name for `UUID.randomUUID` before the method is called?